letsencrypt/tests/test_letsencrypt.py

```diff
@@ -9,7 +9,7 @@ class TestLetsencrypt(TransactionCase):
     post_install = True
 
     def setUp(self):
-        return super(TestLetsencrypt, self).setUp()
+        super(TestLetsencrypt, self).setUp()
 
-    def test_certificate_creation(self):
-        pass
+    def test_http_challenge_creation(self):
+        self.env['letsencrypt'].with_context(DRY_RUN=True).cron()
```

Comment:
you can't do that, this will always fail, because the hostname will end up as localhost and we can't request a certificate for that. We also don't have any real domain to work on.

If there's no fake domain for which the staging api always returns a good certificate, (https://letsencrypt.org/docs/staging-environment/ doesn't look like that), you're just out of luck testing this while talking to any real server.

So either only test parts of the code steered by the `DRY_RUN` flag as the previous implementation did, or (better) get rid of the flag and mock away the acme calls during tests